src/bot/handlers/close.py

```diff
@@ -0,0 +1,31 @@
+from .utils import is_chat_admin, clear_chat
+from ...model import solve
+from telegram.ext import CommandHandler, Filters
+
+# Messages
+ADMINS_ONLY     = 'Ups!!!, solo los administradores pueden usar este comando :('
+ACTIVE          = 'Para cancelar una discusión, primero se debe crear una usando /create'
+CLOSED          = 'Discusión cerrada satisfactoriamente '
+NO_VOTES        = 'No hay votos, por tanto la discusión no puede ser cerrada. Si quiere cancelarla, use /cancel'
+
+def close(update, context):
+    user = update.effective_user.id
+    chat = update.effective_chat.id
+    try:
+        assert is_chat_admin(context.bot, chat, user), ADMINS_ONLY
+        assert context.chat_data.get('active'), ACTIVE
+        assert context.chat_data.get('voters'), NO_VOTES
```

Comment:
```suggestion
        assert any(context.chat_data.get('voters', dict()).values()), NO_VOTES
```
Check if at least one voter has valid votation